sdk/core/amqp-common/rollup.config.js

```diff
@@ -11,6 +11,11 @@ if (!process.env.ONLY_BROWSER) {
 
 if (!process.env.ONLY_NODE) {
   inputs.push(base.browserConfig());
+  inputs.push(base.browserConfig({ production: true }));
```

Comment:
This will result in having 2 copies of browser config. Why do we need 2?
Also, the `browserConfig` takes a single boolean argument. Why are we passing a json object?